service/src/main/resources/quips/complibot/compliments.json

```diff
@@ -229,5 +240,126 @@
     "tags":[],
     "targetableText":"If there was a zombie apocalypse, I'd want resourceful people like [TARGET] in my camp.",
     "targetableSsml":"If there was a zombie apocalypse, I'd want resourceful people like [TARGET] in my camp."
+  },
+  {
+    "quipGroup":"INFECTIOUS_SMILE",
+    "text":"We better call the doctor, because your smile is infectious.",
+    "ssml":"We better call the doctor, because your smile is infectious.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"We better call the doctor, because [TARGET]'s smile is infectious.",
+    "targetableSsml":"We better call the doctor, because [TARGET]'s smile is infectious."
+  },
+  {
+    "quipGroup":"SUNGLASSES_BRIGHT",
+    "text":"They said you were smart, but I didn't know you'd be so bright I'd have to wear sunglasses around you!",
+    "ssml":"They said you were smart, but I didn't know you'd be so bright I'd have to wear sunglasses around you!",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"F5_REFRESHING",
+    "text":"If you were a key on the keyboard, you would be F5, because you're so refreshing!",
+    "ssml":"If you were a key on the keyboard, you would be F5, because you're so refreshing!",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"YOURE_SO_AMAZING",
+    "text":"I think you're so amazing. You aren't even plugged into the wall!",
+    "ssml":"I think you're so amazing. <break /> You aren't even plugged into the wall!",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"YOURE_SO_AMAZING",
+    "text":"I think you're so amazing. Your hard drive never seems to fill up!",
```

Comment:
This series shouldn't start with "I think".  Complibot should just state it as fact.